src/main.cpp

```diff
@@ -110,8 +110,10 @@ QCoreApplication * createApplication(int &argc, char *argv[], unsigned int &acti
     if( (action) & ( 1 | 2) )
         return new QCoreApplication(argc, argv);  // Ah, we're gonna bail out early, just need a command-line application
     else {
+#if defined (Q_OS_WIN32)
         // Force OpenGL use as we use some functions that aren't provided by Qt's OpenGL layer on Windows (QOpenGLFunctions)
         QApplication::setAttribute(Qt::AA_UseDesktopOpenGL);
```

Comment:
This is a fix we've only had to do on Windows, so there's no need to adjust other platforms as well - that'd introduce the possibility of a side-effect.